.reset/db_reset.sql

```diff
@@ -225,8 +225,7 @@ CREATE TABLE `File` (
   KEY `ParentID` (`ParentID`),
   KEY `OwnerID` (`OwnerID`),
   KEY `ClassName` (`ClassName`),
-  KEY `SubsiteID` (`SubsiteID`),
-  FULLTEXT KEY `SearchFields` (`Filename`,`Title`,`Content`)
```

Comment:
These FULLTEXT keys were causing issues of when running the DemoResetTask, so it seems simpler to just remove them since performance won't be a concern at all on a site this small